rimraf.js

```diff
@@ -107,6 +108,9 @@ function rimraf (p, options, cb) {
           if (er.code === "ENOENT") er = null
         }
 
+        if (options.verbose)
+          console.log('deleting', p)
```

Comment:
fwiw, we should use `console.info`

Reply:
Why?  It's just writing a string to stdout, in this case they're 100% equivalent, I almost always use `console.log()` for this purpose.

Reply:
A lot of terminal profiles can be (and are) setup to visually highlight different forms of logs. afaik there is also a screen reader for terminal output which can differentiate between log & info.

Reply:
You may dismiss this suggestion if it isn't aligned with other code here. I did prefix this with fwiw - not a deal breaker for most.

Reply:
They are literally the same function: https://github.com/nodejs/node/blob/main/lib/internal/console/constructor.js#L682

The only way you could highlight different forms of logs is by patching the global `console` object.

They're displayed differently in the browser, but in Node, they're 100% identical, just same function with two different names.